src/SnapshotCodeLens/SnapshotCodeLensProvider.ts

```diff
@@ -20,7 +20,7 @@ export function registerSnapshotCodeLens(enableSnapshotPreviews: boolean) {
 
 class SnapshotCodeLensProvider implements vscode.CodeLensProvider {
   public provideCodeLenses(document: vscode.TextDocument, _token: vscode.CancellationToken) {
-    const snapshots = new Snapshot()
+    const snapshots = new Snapshot(undefined)

```

Comment:
That change is only temporary until [Jest/#6212](https://github.com/facebook/jest/pull/6212) is published.